README.md

```diff
@@ -1,6 +1,6 @@
 ![VPhysics Jolt Logo](assets/cube_base_nobg.png "VPhysics Jolt")
 
-## What is Volt? ⚡
+# What is Volt? ⚡
```

Comment:
? why did this change

Reply:
I think my markdown-lint extension was complaining about it, something about format, no problem.